migrations/sql/V2020.01.09.9.30__update_sand_gravel_operation.sql

```diff
@@ -0,0 +1,22 @@
+ALTER TABLE sand_gravel_quarry_operation
+ADD COLUMN average_overburden_depth_unit_type_code character varying(3),
+ADD COLUMN average_top_soil_depth_unit_type_code character varying(3);
+
+
+ALTER TABLE sand_gravel_quarry_operation
+ADD FOREIGN KEY (average_overburden_depth_unit_type_code) REFERENCES unit_type(unit_type_code) DEFERRABLE INITIALLY DEFERRED,
+ADD FOREIGN KEY (average_top_soil_depth_unit_type_code) REFERENCES unit_type(unit_type_code) DEFERRABLE INITIALLY DEFERRED;
+
+
+UPDATE unit_type set unit = 't' where unit_type_code = 'MTN';
```

Comment:
Ideally I think this column is named "unit" as you've used in the response model, or "short_description" or "display_value" or something along that line as "type" has a meaning in your database that might cause confusion. I'm fine leaving this as is if time constraints prevent a change.

Reply:
I can update this!